onnxscript/rewriter/pattern.py

```diff
@@ -621,10 +623,17 @@ def _make_node(self, op_type: str, inputs: Sequence[ir.Value], kwargs: dict[str,
             assert isinstance(outputs, int)
             num_outputs = outputs
         if num_outputs == 1:
-            return self._tape.op(op_type, inputs=inputs, attributes=kwargs, domain=domain)
-        return self._tape.op_multi_output(
+            value = self._tape.op(op_type, inputs=inputs, attributes=kwargs, domain=domain)
+            if isinstance(outputs, Sequence):
+                value.name = outputs[0]
+            return value
+        values = self._tape.op_multi_output(
```

Comment:
Q: Do we have multi-outputs with pattern.py now? If so, should we remove generic_pattern.py to avoid the potential confusion that users might have?

Reply:
No. This is different (it's about creating nodes with multiple outputs).